docs/DeveloperGuide.md

```diff
@@ -537,10 +537,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * **Mainstream OS**: Windows, Linux, Unix, MacOS
 * **DBMS**: Database Management System, a software providing tools for structural data storage.
-* **CLI**: Command-line interface where you interact with the system using your keyboard
-* **API**: Application Programming Interface, a set of programming code that enables data transmission between one software and another
-* **Abstraction and Cohesion**: Abstraction is hiding all but relevant data in order to reduce complexity and increase efficiency. Cohesion is the degree which elements belong together
-* **Version Control System**: Version control is the practice of tracking and managing changes to software code, and there are many version control systems such as Git, Apache Subversion etc
+* **CLI**: Command-line interface where you interact with the system using your keyboard.
```

Comment:
Perhaps adding a comma behind `Command-line interface, where ...`

Reply:
I will add this in the pr for the DG